client/components/SongRow/index.tsx

```diff
@@ -45,49 +45,20 @@ const SongRow: React.FC<SongRowProps> = (props: SongRowProps) => {
             <li
                 className={
                     props.isCurrentlyPlaying
-                        ? `${style.playing} ${style.songRow} ${props.className}`
-                        : `${style.songRow} ${props.className} `
+                        ? `nowPlaying ${style.songRow} card-clear ${props.className}`
```

Comment:
is `nowPlaying` doing something?

Reply:
```nowPlaying``` is a utility class for consistent site-wide styling